enamlnativecli/templates/app/{{cookiecutter.project_name}}/src/main.py

```diff
@@ -1,6 +1,7 @@
 import os
 import sys
-
+if sys.version_info.major>2:
+    from importlib import reload
```

Comment:
I added this in the pybridge.c bootstrap code so it doesn't need to be done here.

https://github.com/codelv/enaml-native/blob/master/android/src/main/jni/pybridge.c#L302-L303

Reply:
but is not working

Reply:
I will revert the commit, but it is not working.

Reply:
I may have not released the one with this in and it might actually be more clear if it's added here. 

I could revert the other change and put it in here. What are your thoughts?